### PR TITLE
Propagate login and logout across tabs

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -48,6 +50,23 @@ class Masq {
...
+  _listenForLoginOrSignout () {
+    window.addEventListener('storage', async (e) => {
+      if (e.key === CURRENT_USER_INFO_STR) {
+        if (e.newValue === null) {
+          if (this.isLoggedIn()) {
+            await this.signout()
+          }
+        } else if (e.newValue) {
+          await this.connectToMasq()
+          this.eventTarget.dispatchEvent(new Event('logged_in'))
```
Comment (G-Ray) [inline src/index.js]: what about using events named `login`, and `signout` ?

File: .eslintrc.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ module.exports = {
   extends: ['standard'],
   plugins: ['mocha'],
   env: {
+    'browser': true,
```
Comment (G-Ray) [inline .eslintrc.js]: this is needed for the events, window etc ?

File: src/index.js (hunk 1 above)
Comment (sujeebant) [reply src/index.js]: I prefered 'logged_in' to indicate that the event was not triggered when the login is tried but when the login is successful

File: .eslintrc.js (hunk 2 above)
Comment (sujeebant) [reply .eslintrc.js]: yes

**Pull request merged**